docs/references/general/fitacf.md

```diff
@@ -1,2 +1,169 @@
+| *x_w_l* | *m/s* | *[num_pts]* | ***float*** | Spectral width from lamdba fit of XCF | 
+| *x_w_l_e* | *m/s* | *[num_pts]* | ***float*** | Spectral width error from lamdbad fit of XCF | 
+| *x_w_s* | *m/s* | *[num_pts]* | ***float*** | Spectral wdith from sigma fit of XCF | 
+| *x_w_s_e* | *m/s* | *[num_pts]* | ***float*** | Spectral wdith error from sigma fit of XCF | 
+| *phi0* | **None** | *[num_pts]* | ***float*** | Phase determination at lag zero of the ACF | 
+| *phi0_e* | **None** | *[num_pts]* | ***float*** | Phase determination error at lag zero of the ACF | 
+| *elv* | *degrees* | *[num_pts]* | ***float*** | Elevation angle estimate |
+| *elv_low* | *degrees* | *[num_pts]* | ***float*** | Lowest estimated of elevation angle |
+| *elv_high* | *degrees* | *[num_pts]* | ***float*** | Highest estimate of elevation angle |
+| *x_sd_l* | **None** | *[num_pts]* | ***float*** | Standard deviation of lambda fit of XCF|
+| *x_sd_s* | **None** | *[num_pts]* | ***float*** | Standard deviation of sigma fit of XCF |
+| *x_sd_phi* | **None** | *[num_pts]* | ***float*** | Standard deviation of phase fit of XCF |
+
+## File structure
+
+FITACF files are comprised of records that record data nominally every 3 seconds for each integration time.
+Nominally each integration time will change the beam, `bmnum` indicates the beam number. However, some control programs like *camping mode* will stay on one beam for a whole scan. 
+A scan is determined with the `scan` flag by the value. 1 or -1 inidicates the beginning of scan then the value changes to 0 for the rest the scan. When the `scan` value changes from 0 to 1 this indicates the end of the scan. Nominally scans try to fit in a 60 second window. Older FITACF files may show scan times of 120 seconds. Certain *control programs* could change the scan length as well especially newer upcoming radars. During  *normal scan* beams will scan increasing/decreasing from 0
+to 15, however, radars that have more than 16 beams will sometimes start at other beams to meet the 3 second integration time and 60 second scan time. 
+
+!!! Note 
+    *Control programs* can change the beam pattern; `cp` will indicate the *control program* numerical value, and `combf` sometimes contains the *control program's* command/name. 
+
+Uncommonly FITACF records can appear partial meaning there are missing **vector** fields. For example: 
+```bash
+        float   "noise.sky" = 17.799486
+        float   "noise.lag0" = 0.000000
+        float   "noise.vel" = 0.000000
+arrays:
+        short   "ptab" [7]
+        short   "ltab" [19][2]
+        float   "pwr0" [100]
+scalars:
+        char    "radar.revision.major" = 1
+        char    "radar.revision.minor" = 18
+        char    "origin.code" = 0
+
+```
+Partial records are a legacy behavior of RST that happens because of potentially "bad" data, or no data was collected, making `slist` empty. 
+Currently, this is an ongoing investigation. 
```

Comment:
```suggestion
Partial records are a legacy behavior of the DMAP file format resulting from potentially "bad" data or missing data, which produces an empty `slist`. The Data Analysis Working Group is investigating this behavior.
```

Reply:
@egthomas Yes, I think that in most cases the partial records are due to an empty slist (probably including the example provided here). But @mts299 has reported that fields can be missing for other reasons (e.g. https://github.com/SuperDARN/rst/issues/230#issuecomment-567244431). I think we should rephrase this line to better emphasise that most of the time this is the nominal behaviour.

Reply:
I have seen them in MAP files as well. Are we okay with what @ecbland suggested? I can commit it.

Reply:
@mts299 I've edited my suggestion to better emphasise that the partial records are the nominal behaviour for the DMAP file format. I realise this is not the whole story, but with our current understanding of the issue we probably can't add anything else to the documentation that would be helpful to users.